simulator/memory/t/unit_test.cpp

```diff
@@ -147,10 +147,10 @@ TEST_CASE( "Func_memory: Host_Guest_Memcpy_1b")
     Byte read_data_1{ 0xFF};
 
     // Write
-    CHECK( func_mem->memcpy_host_to_guest_noexcept( dataSectAddr, &write_data_1, 1) == 1);
+    CHECK( func_mem->memcpy_host_to_guest_noexcept( dataSectAddr, &write_data_1, 1) == 0x1);
```

Comment:
Wait a minute, my comment was related to the different line...

Reply:
Oh, I see. So as I understand it, we need to  use `0xA5` directly instead of write_data_1 in line 153